internal/observability/event/event_audit.go

```diff
@@ -14,7 +14,7 @@ const auditVersion = "v0.1"
 type auditEventType string
 
 const (
-	apiRequest auditEventType = "APIRequest" // ApiRequest defines an API request audit event type
+	ApiRequest auditEventType = "APIRequest" // ApiRequest defines an API request audit event type
```

Comment:
Is this used outside of Boundary? Just wondering if we want `APIRequest` vs. `API_Request` or `ApiRequest`

Reply:
Currently, It's slated to be in audit events and I was choosing to match Consul and Nomad's value.